src/SFA.DAS.LevyAggregationProvider.Worker.UnitTests/Providers/LevyAggregationTests/WhenIAggregateLevyData.cs

```diff
@@ -223,8 +224,8 @@ public void ThenTheTotalIncludesTheTopUpValueAndTheEnglishPercentage()
             var actual = _levyAggregator.BuildAggregate(actualData);
 
             //Assert
-            var expectedTotal = (actualData.Data[0].LevyDueYtd * actualData.Data[0].EnglishFraction) + (actualData.Data[0].TopUp * actualData.Data[0].EnglishFraction);
-            Assert.AreEqual(expectedTotal, actual.Data[0].Balance);
+            var expectedTotal = Math.Round(actualData.Data[0].LevyDueYtd * actualData.Data[0].EnglishFraction,2) + Math.Round(actualData.Data[0].TopUp * actualData.Data[0].EnglishFraction,2);
```

Comment:
Could you not just have round the final value of expectedTotal rather than round the two values and then have to round them again afterwards?